src/Meshes/mesh.ts

```diff
@@ -3449,12 +3450,12 @@ export class Mesh extends AbstractMesh implements IGetSetVerticesData {
         }
 
         // Morph targets
-        if (parsedMesh.morphTargetManagerId > -1) {

```

Comment:
why this one? what if id is === 0?

Reply:
ah, thanks for catching that, I missed copying over my change @L3458

Reply:
I'll also add a `null` check as well

Reply:
I didn't realize that `MorphTargetManagerId` can only be int :), reverting that change